banyand/index/index.go

```diff
@@ -108,7 +123,7 @@ func (s *service) Insert(series common.Metadata, shardID uint, field *Field) err
 	}
 	objects, ok := sd.meta[field.Name]
 	if !ok {
-		return ErrUnknownField
+		return nil
```

Comment:
Could you record a debug level message to indicates which fields are ignored?

Reply:
Added